extensions/replication/utils/QueueEntry.js

```diff
@@ -135,6 +135,18 @@ class QueueEntry {
         };
     }
 
+    getLocation() {
+        return this.objMd.location;
+    }
+
+    getPartNumber(location) {
+        return Number(location.dataStoreETag.split(':')[0], 10);
```

Comment:
I think you have to use `Number.parseInt()` to provide the base, the constructor seems to only take one argument.